### ISSUE TITLE
Shared ruleset for mitigating bot attacks

### ISSUE THREAD

**Issue by v3gard** [equinor/radix-platform#759]
**Is your feature request related to a problem? Please describe.**
Bot attacks are commonly spotted on public HTTP endpoints. A common way to mitigate these is to configure [fail2ban ](https://www.digitalocean.com/community/tutorials/how-to-protect-an-nginx-server-with-fail2ban-on-ubuntu-20-04)to monitor the access and/or error logs and thus block the source IP address in the local firewall (iptables).

e.g from our application hosted in Radix.
![image](https://user-images.githubusercontent.com/343078/192288810-ec9f71af-b763-401d-9848-6f88f2d16bc5.png)

**Describe the solution you'd like**
A [shared fail2ban configuration](https://www.digitalocean.com/community/tutorials/how-to-protect-an-nginx-server-with-fail2ban-on-ubuntu-20-04) or user agent [blacklist ](https://gist.github.com/hans2103/733b8eef30e89c759335017863bd721d)that can be applied for the nginx instance in the pod running in a given environment.

**Describe alternatives you've considered**
Handling the configuration on our own apps. That will also work, but this is a general problem for all cloud native apps that requires a general solution.

**Additional context**
N/A

**Comment by oyron** [comment]
Thank you for reporting this issue, and sorry for the long response time. We will look into this shortly and get back to you.

**Comment by oyron** [comment]
We have discussed the proposed feature and decided not to prioritize this. We agree that this is a common problem that ideally should be solved generally. However, with limited capacity and the fact that none of the app teams have reported this as an issue, there are other features that are more imporant to implement. Also, we see that there is a risk that a traffic-based, blocklist-approach could lead to a DOS scenario, that we consider to be a bigger risk than the unwanted traffic. Closing the issue, but let us know if this is starting to become a blocker.
